sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/TxOutputsView.kt

```diff
@@ -69,7 +69,9 @@ internal class TxOutputsView(private val sqliteDatabase: SupportSQLiteDatabase)
                 val toAccountIndex = it.getColumnIndex(TxOutputsViewDefinition.COLUMN_BLOB_TO_ACCOUNT)
                 val toAddressIndex = it.getColumnIndex(TxOutputsViewDefinition.COLUMN_STRING_TO_ADDRESS)
 
-                if (!it.isNull(toAccountIndex)) {
+                if (!it.isNull(toAddressIndex)) {
```

Comment:
The reordering of the condition means that the address is preferred before the account, which might not be correct.